### DISCUSSION TITLE
Better device management in users tab

### DISCUSSION THREAD

**Discussion by ConGp20** [rustdesk/rustdesk-server-pro#214] [Ideas]
I'd love to have the feature to only show all unassigned devices f.ex. to admins. So that if you have devices that are not assigned to any user, will not be shown in clients with f.ex. a customer account signed in. That would be nice because i have accounts for customers that should have an adressbook and acess to their company-devices. But if i have other costumer devices added without an account assigned these should not be shown as standard.

I think this is the same problem, that i have:
https://github.com/rustdesk/rustdesk-server-pro/discussions/175#discussion-5952239

**Comment by rustdesk** [comment]
@21pages follow it up and implement it, higher priority than hardware codecs.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk-server-pro/releases/tag/1.3.5

**Comment by ZetaWaves** [comment]
@rustdesk, is there a way to make a separate feature under server settings to allow connections to everyone, but just disable the user from seeing all the machines? 

I want user's to be able to connect to anyone, but I don't want them to have a list of all machines connected to the server once they're logged in.
